source/refined-github.css

```diff
@@ -156,3 +156,10 @@ pr-branches
 .file-header .file-info clipboard-copy {
 	display: inline-block;
 }
+
+/* Limit width of commit title to 72 characters */
+.rgh-monospace-textareas :is(#merge_title_field, #commit-summary-input) {
```

Comment:
My bad, I didn't notice the review request.

This leading `.rgh-monospace-textareas` should be removed since no feature is adding the class anymore.